operators/src/source/gdal_source/mod.rs

```diff
@@ -701,21 +830,37 @@ where
         }
     };
 
-    let output_geo_transform = tile_info.tile_geo_transform();
-    let dataset_grid_bounds =
-        dataset_geo_transform.spatial_to_grid_bounds(&dataset_intersection_area);
+    let tile_geo_transform = tile_info.tile_geo_transform();
+
+    let gdal_read_window =
+        gdal_dataset_geotransform.spatial_partition_to_read_window(&dataset_intersection_area);
+
+    let is_y_axis_flipped = tile_geo_transform.y_pixel_size().is_sign_negative()
```

Comment:
maybe use the "y_axis_is_increasing" method instead

Reply:
and: why is do you do the comparison? I don't understand it.

Reply:
you mean this:     `let is_y_axis_flipped = tile_geo_transform.y_pixel_size().is_sign_negative()!= gdal_dataset_geotransform.y_pixel_size.is_sign_negative();` ? This is how we find out if the data is stored up-side-down. The sign of the y-axis in the geotransform indicates how the pixels are ordered. If the order is not the same it is up-side-down.

Reply:
but in Geo Engine the origin of a tile is always in the top left corner or not? Then tile_geo_transform y_pixelsize should always be negative and we only need to look at the dataset_geo_transform?

And what about the case where the CRS axis are flipped?

Reply:
Well i would argue that there might be a strange SRS where the y-axis is pointing down. Then the pixel size in the GeoTransform we use for our context will be positive. And there might still be a tiff where the datas geotransform has a pixel size with a negative sign

Reply:
still: why do you need to check the tile_geo_transform in order to determine that the y_axis of the tiff is flipped?

Reply:
the GeoTransform of the tile is what we want and the GEoTransform of the data is what we get. If they don't have the same sign the data is flipped. Currently it will always be negative for the tile GeoTransform but do you really hurt to check that? We have to check the sign for the data GeoTransform anyway.